### PR TITLE
Updated instructions in playbooks to build from source

### PR DESCRIPTION
**Pull request by arilivigni**
- Added git tags and checkouts before building origin and openshift-ansible
- Created project dictionary with name and version
- Updated JJB to reflect the new project name

### CODE REVIEW

File: playbooks/openshift/roles/bfs/tasks/build_openshift-ansible_repo.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+- name: set latest tag from git tag
```
Comment (herlo) [inline playbooks/openshift/roles/bfs/tasks/build_openshift-ansible_repo.yml]: This doesn't look like it's building the openshift-ansible repo. Maybe a more semantic name?

Comment (arilivigni) [inline playbooks/openshift/roles/bfs/tasks/build_openshift-ansible_repo.yml]: Again troys instructions for origin this was missing from producing versioned rpms we call tito in a separate task

File: playbooks/openshift/roles/bfs/tasks/build_origin_repo.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+- debug:
```
Comment (arilivigni) [inline playbooks/openshift/roles/bfs/tasks/build_origin_repo.yml]: Again troys instructions for origin this was missing from producing versioned rpms

**Comment by herlo** [comment]
:+1:

**Pull request merged**